### ISSUE TITLE
What happens if we rename "staleLabel" in the "daysUntilClose" period

### ISSUE THREAD

**Issue by ITAYC0HEN** [probot/stale#288]
Hey, thanks for the great work!

When setting up the bot, [we chose](https://github.com/radareorg/radare2/pull/16924/files) "wontfix" as the "staleLabel", and upon merging it did its magic and commented on 100+ issues. Now, few hours later, we figured that "wontfix" wasn't a good decision for "staleLabel" and [we changed](https://github.com/radareorg/radare2/pull/17085) it to "stale".

Now, we still have 100+ open "wontfix" labeled issues that we would like to rename to "stale" via Github labels editor.

The question is, if we will rename the label - what will be the circumstances?
If we won't rename the label, after changing "staleLabel" already, what will be the circumstances?

The bottom line is, that we want StaleBot to work with "stale" label and not the mistake we did when we chose "wontfix". All this, while keeping track of the 100+ issues the bot already marked.

How do you suggest we act?
